packages/runtime/jest.config.js

```diff
@@ -0,0 +1,194 @@
+/*
```

Comment:
https://github.com/webzard-io/meta-ui/blob/main/packages/core/jest.config.js

We can use the general config as the base config.

Reply:
OK. And I found some problem with ts-jest, so I follow the documentation to add a babel.config.js instead of ts-jest.